virtcontainers/network.go

```diff
@@ -810,6 +934,40 @@ func unBridgeNetworkPair(endpoint Endpoint) error {
 	return nil
 }
 
+func tcRemoveNetwork(endpoint Endpoint) error {
```

Comment:
@amshinde you need to remove the ingress qdisc you added to the veth/incoming interface. As part of the cleanup you should remove that too.

So you need to undo what do you did before

```
	if err := addQdiscIngress(attrs.Index); err != nil {
		return err
	}
```

Reply:
@mcastelino Have addressed this. PTAL.